source/common/network/connection_impl_base.h

```diff
@@ -8,6 +8,19 @@
 namespace Envoy {
 namespace Network {
 
+/**
+ * Utility functions for the connection implementation base class.
+ */
+class ConnectionImplBaseUtility {
```

Comment:
I'm not sure if an additional class is adding any value, did you consider just adding this as a static function on `ConnectionImplBase` itself?

Reply:
My intention was to be consistent with [`ConnectionImplUtility`](https://github.com/envoyproxy/envoy/blob/master/source/common/network/connection_impl.h#L39) but I can also just make it a static function.